cypress/plugins/testDataProcessor.js

```diff
@@ -102,45 +102,22 @@ function processSetResponseJson(setResponseJsonData) {
 }
 
 /**
- *  @function processErrorContentJson
- *  processErrorContentJson function will perform following operations
+ *  @function getErrorContentObjectJson
+ *  getErrorContentObjectJson function will perform following operations
  *  - Iterate over each key in the provided JSON
```

Comment:
This description no longer reflects what is happening in the function.

Reply:
addressed the comment